### PR TITLE
FEAT: Added husky precommit hooks

### PR DESCRIPTION
**Pull request by anisharaz**
Added huskey to run linting , formatting & build test before commiting change. #80

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -0,0 +1,3 @@
+npm run format
+npm run lint
+npm run build
```
Comment (Fryingpannn) [inline .husky/pre-commit:3]: We use yarn, wouldn't this be yarn instead of npm?

Comment (anisharaz) [reply .husky/pre-commit]: absolutely can be , changed in the next commit

**Comment by shrix1** [comment]
Good to go

**Review by ICEPrey — APPROVED** [review]

**Comment by anisharaz** [comment]
I hope its ready to merge @Fryingpannn

File: .husky/pre-commit [hunk 2]
```diff
@@ -0,0 +1,3 @@
+yarn format
+yarn lint
+yarn build
```
Comment (Fryingpannn) [inline .husky/pre-commit:3]: Potentially remove yarn build in future. lets try it for now

**Review by Fryingpannn — APPROVED** [review]

**Pull request merged**